lib/improver/spotdata/neighbour_finding.py

```diff
@@ -122,7 +122,7 @@ def process(self, cube, sites, ancillary_data,
 
         """
         if self.method == 'fast_nearest_neighbour':
-            if 'orography' in ancillary_data.keys():
+            if 'orography' in list(ancillary_data.keys()):
```

Comment:
just need `if 'orography' in ancillary_data`

Reply:
Done.